scylla-server-rust/tests/data_service_test.rs

```diff
@@ -0,0 +1,128 @@
+#[path = "test_utils.rs"]
+mod test_utils;
+
+use prisma_client_rust::QueryError;
+use protobuf::SpecialFields;
+use scylla_server_rust::{
+    controller::data_controller::DataSend,
+    serverdata::ServerData,
+    services::{data_service, data_type_service, node_service, run_service},
+};
+use test_utils::cleanup_and_prepare;
+
+const DATA_TYPE_NAME: &str = "test";
+
+#[tokio::test]
+async fn test_data_service() -> Result<(), QueryError> {
+    let db = cleanup_and_prepare().await?;
+
+    run_service::create_run_with_id(&db, 0, 0).await?;
+    node_service::upsert_node(&db, DATA_TYPE_NAME.to_owned()).await?;
+    data_type_service::upsert_data_type(
+        &db,
+        DATA_TYPE_NAME.to_owned(),
+        "joe_mama".to_owned(),
+        DATA_TYPE_NAME.to_owned(),
+    )
+    .await?;
+    data_service::get_data(&db, DATA_TYPE_NAME.to_owned(), 0).await?;
+
+    Ok(())
```

Comment:
theres no asserts here?

Reply:
or is this just a smoke test?

Reply:
its a "doesnt return err test", how it was originally written I believe.

Reply:
yeah thats called a smoke test